sklearn/linear_model/tests/test_base.py

```diff
@@ -306,9 +311,10 @@ def test_linear_regression_positive_multiple_outcome(random_state=0):
     assert_allclose(np.vstack((y_pred, y_pred)).T, Y_pred)
 
 
-def test_linear_regression_positive_vs_nonpositive():
+def test_linear_regression_positive_vs_nonpositive(global_random_seed):
     # Test differences with LinearRegression when positive=False.
-    X, y = make_sparse_uncorrelated(random_state=0)
+    random_state = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all